materialsLibrary/src/normal/normal.fragment.fx

```diff
@@ -97,4 +97,8 @@ void main(void) {
 #include<fogFragment>
 
 	gl_FragColor = color;
+
+#ifdef IMAGEPROCESSINGPOSTPROCESS

```

Comment:
this should be replaced with a `#include <linearTransform.fx>`, but when doing this, a weird 301 redirection to an invalid URL happens in the playground. Any idea why ?

Reply:
discussed offline